vector/src/main/java/im/vector/app/features/popup/PopupAlertManager.kt

```diff
@@ -235,7 +235,8 @@ class PopupAlertManager @Inject constructor() {
                         setIcon(it)
                     }
                     alert.actions.forEach { action ->
-                        addButton(action.title, R.style.Widget_Vector_Button_Text_Alerter) {
+                        val textStyle = alert.textStyle ?: R.style.Widget_Vector_Button_Text_Alerter
```

Comment:
Should be possible to override this theme instead of create a new one for the textbutton ?